crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -375,6 +375,7 @@ impl ServerBuilder {
         let mut builder = ServerBuilder::new(state);
 
         let name_service_config = config.service.name_service.clone();
+        let dot_move_config = config.service.move_registry.clone();
```

Comment:
nit: less important but may become confusing with time.

```suggestion
        let move_registry_config = config.service.move_registry.clone();
```